conslogging/conslogging.go

```diff
@@ -245,6 +247,20 @@ func (cl ConsoleLogger) PrintBytes(data []byte) {
 	}
 }
 
+// Debugf prints formatted text to the console when verbose flag is set.
+func (cl ConsoleLogger) Debugf(format string, args ...interface{}) {
```

Comment:
Wondering if we should call this something like `Verbosef` or `VerbosePrintf`. The word debug can be overloaded when talking about printing: is it for us trying to debug Earthly, or is it for the user trying to debug their Earthfile.

Reply:
that's a good point. I was initially thinking that perhaps one day we could even extend it to be more like syslog and have levels like: Debug, Info, Warn, Error, then rather than having verbose=true/false, we could have more control over setting the levels, but that doesn't address the `-V` vs -D` flags we currently have.